script/ConfigurePairs.s.sol

```diff
@@ -0,0 +1,195 @@
+        sphinxConfig.projectName = "nana-core";
+        sphinxConfig.mainnets = ["ethereum", "optimism", "base", "arbitrum"];
+        sphinxConfig.testnets = ["ethereum_sepolia", "optimism_sepolia", "base_sepolia", "arbitrum_sepolia"];
+    }
+
+    function run() public {
+        // Get the deployment addresses for the swap terminal contracts for this chain.
+        swapTerminal = SwapTerminalDeploymentLib.getDeployment(
+            vm.envOr("NANA_SWAP_TERMINAL_DEPLOYMENT_PATH", string("deployments/"))
+        );
+
+        deploy();
+    }
+
+    function deploy() private sphinx {
+        // Configure pairs for the swap terminal.
+        // DAI/ETH (0.05%)
+        configurePairFor(
+            ETHEREUM_MAINNET,
+            0x6B175474E89094C44Da98b954EedeAC495271d0F,
+            IUniswapV3Pool(0x60594a405d53811d3BC4766596EFD80fd545A270),
+            10 minutes,
+            200 // 2% slippage tolerance
+        );
+        // USDC/ETH (0.05%)
+        configurePairFor(
+            ETHEREUM_MAINNET,
+            0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48,
+            IUniswapV3Pool(0x88e6A0c2dDD26FEEb64F039a2c41296FcB3f5640),
+            2 minutes,
+            100 // 1% slippage tolerance
+        );
+        // USDT/ETH (0.05%)
+        configurePairFor(
+            ETHEREUM_MAINNET,
+            0xdAC17F958D2ee523a2206206994597C13D831ec7,
+            IUniswapV3Pool(0x11b815efB8f581194ae79006d24E0d814B7697F6),
+            2 minutes,
+            100 // 1% slippage tolerance
+        );
+
+        // ETH/DAI (0.3%)
+        configurePairFor(
+            ARBITRUM_MAINNET,
+            0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1,
+            IUniswapV3Pool(0xA961F0473dA4864C5eD28e00FcC53a3AAb056c1b),
+            5 minutes,
+            200
+        );
+        // ETH/USDC (0.05%)
+        configurePairFor(
+            ARBITRUM_MAINNET,
+            0xaf88d065e77c8cC2239327C5EDb3A432268e5831,
+            IUniswapV3Pool(0xC6962004f452bE9203591991D15f6b388e09E8D0),
+            5 minutes,
+            100
+        );
+        // ETH/USDT (0.05%)
+        configurePairFor(
+            ARBITRUM_MAINNET,
+            0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9,
+            IUniswapV3Pool(0x641C00A822e8b671738d32a431a4Fb6074E5c79d),
+            5 minutes,
+            100
+        );
+
+        // ETH/DAI (0.3%)
+        configurePairFor(
+            OPTIMISM_MAINNET,
+            0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1,
+            IUniswapV3Pool(0x03aF20bDAaFfB4cC0A521796a223f7D85e2aAc31),
+            30 minutes,
+            200 // 2% slippage tolerance
+        );
+        // USDC/ETH (0.05%)
+        configurePairFor(
+            OPTIMISM_MAINNET,
+            0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85,
+            IUniswapV3Pool(0x1fb3cf6e48F1E7B10213E7b6d87D4c073C7Fdb7b),
+            30 minutes,
+            200 // 2% slippage tolerance
+        );
+        // USDT/ETH (0.05%)
```

Comment:
just double checking: does it matter than in your comments some of these lead with ETH and some lead with the paired token? i.e. USDC/ETH vs ETH/DAI